html/api/editcontact.php

```diff
@@ -23,13 +23,15 @@
     }
 	else
 	{
-		$sql = "UPDATE contacts SET firstName='". $inData["firstName"] . "', lastName='" . $inData["lastName"] ."', phone='". $inData["phone"] ."', email='" . $inData["email"] . "', dateCreated='". date("Y/m/d") ."' WHERE id= ". intval($inData["id"]);
+		$sql = "UPDATE contacts SET firstName='". $inData["firstName"] . "', lastName='" . $inData["lastName"] ."', phone='". $inData["newPhone"] ."', email='" . $inData["email"] . "', dateCreated='". date("Y/m/d") ."' WHERE phone= '" . $inData["oldPhone"] . "'";
```

Comment:
You gotta make sure it matches the userId as well

Reply:
done